code_schemes/current_practises.json

```diff
@@ -0,0 +1,71 @@
+{
+    "SchemeID": "Scheme-8baa86c3",
+    "Name": "Current practices used to improve soil quality",
```

Comment:
Is this a follow-up question?

Wondering if we can have a more succinct scheme name, but don't have any suggestions.

Reply:
Yes, this is a follow up question.  Name updated to Current practices